src/QualificationRunner.Core/QualificationRunnerConfiguration.cs

```diff
@@ -49,7 +49,7 @@ private string getRegistryValueForRegistryPathAndKey(string openSystemsPharmacol
       {
          try
          {
-            return (string) Registry.GetValue($@"HKEY_LOCAL_MACHINE\SOFTWARE\{openSystemsPharmacology}{Version}", installDir, null);
+            return (string) Registry.GetValue($@"HKEY_LOCAL_MACHINE\SOFTWARE\{openSystemsPharmacology}{Major}", installDir, null);

```

Comment:
Now trying to find the version at the right place. 
I also updated the build to use the same structure as for v10